docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+    enable dynamic control allocation on older firmware, so it is possible to test using the actuator tab in that case as well, but it is not strictly necessary.
+
+Older PX4 Firmware (v1.13.2 and earlier)
+#########################################
+Under the Motors tab in QGroundControl, there are sliders that can test if the modules will spin, and if the flight controller is configured to use DroneCAN properly. 
+If DroneCAN is enabled, this test should send DroneCAN commands to control the modules.
+
+To run the test, arm your safety switch if you have one, and click the horizontal slider at the bottom of the window to enable the motors. Make sure the Vertiq module 
+is powered up and connected to the same CAN bus as the flight controller.
+
+Then move the slider corresponding to the motor number of the module you wish to test. E.g. to test a module with an ESC index of 0, move slide 1. If configured correctly, 
+the motor with the corresponding index should start spinning as you move the slider.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/px4_dronecan_motor_test_sliders.png
+    :align: center
+    :width: 60%
+    :alt: Motor Test Sliders
+
+    Motor Test Sliders in Motors Tab
+
+Newer PX4 Firmware (v1.13.3 and later)
+#########################################
+Under the Actuators tab in QGroundControl, there are sliders in the Actuator Testing section that can be used to test if the flight controller and module have been
+successfully integrated.
+
+To run the test, arm your safety switch if you have one, and click the horizontal slider at the top of the Actuator Testing section to enable the motors. Make sure the Vertiq module 
+is powered up and connected to the same CAN bus as the flight controller.
+
+Then move the slider corresponding to the motor number of the module you wish to test. If configured correctly, the corresponding motor should spin as you move the slider. An
+example of using this test is shown below.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/qgc_actuator_testing.png
+    :align: center
+    :width: 40%
+    :alt: Actuator Testing Sliders
+
+    Actuator Testing Sliders in Actuators
+
+MAVLink Console
+****************
+The MAVlink console can also be used to check if your motor is sending out the expected status messages. To open the MAVLink console, on the home screen from QGroundControl 
+click on the QGroundControl logo and select Analyze Tools from the Select Tool menu, as shown below.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/qgc_select_tools.png
+    :align: center
+    :width: 60%
+    :alt: Select Tool Menu
+
+    Select Tool Menu in QGroundControl
+
+In Analyze Tools, select the MAVLink Console. This makes it possible to execute various DroneCAN related commands through the flight controller. Confirm that the Vertiq module 

```

Comment:
such as? are these various dronecan related commands mentioned below?

Reply:
Yes? In the two sections immediately below this I walk through using them